charts/flyte-core/templates/admin/deployment.yaml

```diff
@@ -111,6 +111,15 @@ spec:
         - containerPort: 8088
         - containerPort: 8089
         - containerPort: {{ .Values.configmap.adminServer.flyteadmin.profilerPort }}
+        readinessProbe:
+          exec:
+            command: [ "sh", "-c", "reply=$(curl -s -o /dev/null -w %{http_code} http://127.0.0.1:8088/healthcheck); if [ \"$reply\" -lt 200 -o \"$reply\" -ge 400 ]; then exit 1; fi;","grpc_health_probe", "-addr=:8089"]
```

Comment:
I'm not quite sure what does this go.. is this a command because that's checking for grpc health? If it's just checking for the http endpoint health, can this be an http probe instead? I think the gke ingress can automatically pickup http probes if I recall correctly.
Or maybe it's it possible to have a probe for each exposed port? Then we can do http for 8088 and a command for 8089?

Reply:
The script has two parts
* Checks the health REST endpoint and checks the returns code if between 200 and 400 and passes the check
* Uses the grpc health probe directly

The result combines the two and fails if any of the checks fail 
We can't have probe for each exposed port unfortunately https://github.com/kubernetes/kubernetes/issues/37218
And the suggested way was to use a script